group1/communication/node/main.js

```diff
@@ -173,6 +318,25 @@ var messages = {
                 });
         }
 };
+var indvroom={
+    'save':function (creator_id, second_user) {
+           indvroom_template={
+               data:{
+                   'users':creator_id+'-'+second_user,
+                   'create_user':util.format('http://localhost:8000/api/users/%s/', creator_id),
+                   'second_user':second_user,
+               },
+               headers: { 'Content-Type': 'application/json' }
+           };
+           client.post("http://localhost:8000/api/indvrooms/", indvroom_template, function (data, response) {
+                    // parsed response body as js object
+                    console.log('indvroom created ' + data.id);
```

Comment:
Without this logging statement, this block is empty.  There is nothing else that should go in this block?  Is the log statement useful outside of debugging during development?  Are the commented lines here necessary?  In general, I think comments are very useful.  These comments though are not clear to me what they are commenting.  What is the "parsed response body as js object"?  The commented lines on 334-335 seem like they are not being used anymore and if so can be removed.  If neither the logging statement nor the other commented lines are necessary and you remove them, you should write a comment here indicating why this block is empty.